crds/testsuite-json-schema.yaml

```diff
@@ -0,0 +1,120 @@
+  skipDelete:
+    description: If set, do not delete the resources after running the tests (implies SkipClusterDelete).
+    type: boolean
+    default: false
+  skipClusterDelete:
+    description: If set, do not delete the mocked control plane or kind cluster.
+    type: boolean
+    default: false
+  timeout:
+    description: Override the default timeout of 30 seconds (in seconds).
+    type: integer
+    default: 30
+  parallel:
+    description: The maximum number of tests to run at once.
+    type: integer
+    default: 8
+  artifactsDir:
+    description: The directory to output artifacts to (current working directory if not specified).
+    type: string
+    default: "."
+  commands:
+    description: Commands to run prior to running the tests.
+    type: array
+    items:
+      description: The Command object is used to enable running commands in tests
+      type: object
+      properties:
+        command:
+          description: The command and argument to run as a string.
+          type: string
+        script:
+          description: |
+            Allows a shell script to run 
+            - namespaced and command should not be used with script. 
+            - namespaced is ignored and command is an error. 
+            - env expansion is depended upon the shell but ENV is passed to the runtime env.
+          type: string
+        namespaced:
+          description: |
+            If set, the --namespace flag will be appended to the command with the namespace to use 
+            (the test namespace for a test step or "default" for the test suite).
+          type: boolean
+        ignoreFailure:
+          description: If set, failures will be ignored.
+          type: boolean
+        background:
+          description: |
+            If this command is to be started in the background. 
+            These are only support in TestSuites.
+          type: boolean
+        skipLogOutput:
+          description: |
+            If set, the output from the command is not logged. 
+            Useful for sensitive logs or to reduce noise.
+          type: boolean
+        timeout:
+          description: Override the TestSuite timeout for this command (in seconds).
+          type: integer
+  kindContainers:
+    description: List of Docker images to load into the KIND cluster once it is started.
+    type: array
+    items:
+      type: string
+    default: []
+  reportFormat:
+    description: | 
+      Determines the report format. If empty, no report is generated. One of: JSON, XML.
+    type: string
+  reportName:
+    description: The name of report to create. This field is not used unless reportFormat is set.
+    default: "kuttl-test"
+    type: string
+  namespace:
+    description: |
+      The namespace to use for tests. This namespace will be created if it does not exist 
+      and removed if it was created (unless skipDelete is set). 
+      If no namespace is set, one will be auto-generated.
+   type: string
```

Comment:
glad I looked deeper... this `type:` is at the wrong indentation.   I can't push to your remote fork.   I'll merge and update on another PR.  thanks for the contribution!

Reply:
thanks. I've added a commit with the change. Feel free to squash it, or if you prefer to merge as is and clean up after. Thanks again for your review and feedback.